docs/docs/README.md

```diff
@@ -0,0 +1,51 @@
+# Atlassian Data Center Helm Charts
+
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/atlassian-data-center)](https://artifacthub.io/packages/search?repo=atlassian-data-center)
+[![Atlassian license](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/CONTRIBUTING.md) 
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-helm-charts/blob/master/CONTRIBUTING.md) 
+[![Maven unit tests](https://github.com/atlassian-labs/data-center-helm-charts/actions/workflows/maven.yml/badge.svg)](https://github.com/atlassian-labs/data-center-helm-charts/actions/workflows/maven.yml)
+
+This project contains [Helm charts](https://helm.sh/){.external} for installing Atlassian's [Jira Data Center](https://www.atlassian.com/enterprise/data-center/jira){.external}, [Confluence Data Center](https://www.atlassian.com/enterprise/data-center/confluence){.external}, and [Bitbucket Data Center](https://www.atlassian.com/enterprise/data-center/bitbucket){.external} on Kubernetes. 
```

Comment:
If the link is external, I've appended `{.external}` class to it so it will display a small icon signifying that link goes outside of the docs.